src/nvim/testdir/test_signs.vim

```diff
@@ -6,6 +6,8 @@ endif
 
 source screendump.vim
 
+set noswapfile
```

Comment:
It's not present in the original vim patch, is this necessary? In my environment, the test passes without it.

Reply:
I don't pass... Because

```
Found errors in Test_sign():
Caught exception in Test_sign(): Vim(new):E325: ATTENTION @ function RunTheTest[44]..Test_sign, line 42
Found errors in Test_sign_group():
Caught exception in Test_sign_group(): Vim(split):E325: ATTENTION @ function RunTheTest[44]..Test_sign_group, line 207
Found errors in Test_sign_jump_func():
Caught exception in Test_sign_jump_func(): Vim(edit):E325: ATTENTION @ function RunTheTest[44]..Test_sign_jump_func, line 5
```

I think swapfile is on in neovim default.

Reply:
> I don't pass... 

I think that's because the swap file is still there when the test ended abnormally by `SEGV`. Therefore, instead of `setting noswapfile` here, you should delete any remaining swap files in your environment.

Reply:
OK.  I have removed it.